tests/codebuild/inject_tests.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+source run_test.sh
+
+function inject_all_variables
```

Comment:
++ great refactor, can add comment so that file is self-explanatory? Why are we injecting variables, for example?